pkg/kudoctl/resources/dependencies/resolve.go

```diff
@@ -144,6 +144,16 @@ func indexOf(dependencies *[]Dependency, dependency *Dependency) int {
 	return -1
 }
 
+// fullyQualifiedName formats a TaskSpec for human readable consumption.
 func fullyQualifiedName(kt kudoapi.KudoOperatorTaskSpec) string {
-	return fmt.Sprintf("%s-%s", kudoapi.OperatorVersionName(kt.Package, kt.OperatorVersion), kt.AppVersion)
+	operatorVersion := kt.OperatorVersion
+	if operatorVersion == "" {
```

Comment:
the dependency can really depend on ANY OV? 🤔

Reply:
I would assume so. I think it's perfectly fine to depend on "package: zookeeper, appVersion: 2.5, operatorVersion: any"